projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -774,6 +777,21 @@ describe('IgxDateRangePicker', () => {
 
                 expect((dateRange as any).calendar.selectedDates.length).toBeGreaterThan(0);
             }));
+
+            it('should set correct valid state when the form group is disabled', () => {
```

Comment:
```suggestion
            it('should set initial validity state when the form group is disabled', () => {
```

Reply:
Also, this test only covers the single input range picker, we should also have the same test scenario for when the picker has projected inputs.